errors/daemon.go

```diff
@@ -226,12 +226,21 @@ var (
 	})
 
 	// ErrorCodeIPCRunning is generated when we try to join a container's
-	// IPC but its not running.
+	// IPC but it's not running.
 	ErrorCodeIPCRunning = errcode.Register(errGroup, errcode.ErrorDescriptor{
 		Value:          "IPCRUNNING",
 		Message:        "cannot join IPC of a non running container: %s",
 		Description:    "An attempt was made to join the IPC of a container, but the container is not running",
-		HTTPStatusCode: http.StatusInternalServerError,
+		HTTPStatusCode: http.StatusConflict,
+	})
+
+	// ErrorCodeIPCRestarting is generated when we try to join a container's
+	// IPC but it's restarting.
+	ErrorCodeIPCRestarting = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
Maybe you are right, `ErrorCodeContainerRestarting` could work. at least user would know they are trying to join a restarting container.

I'll change as you suggested, thank you for your suggestion! @coolljt0725 :smile: